plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestingIgniteContainer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ignite;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+
+public class TestingIgniteContainer
+        extends JdbcDatabaseContainer<TestingIgniteContainer>
+{
+    private static final String IGNITE_IMAGE = "apacheignite/ignite:2.8.0";
+    private static final String IGNITE_SQL_MERGE_TABLE_MAX_SIZE = "20000";
+    public static final Integer HTTP_PORT = 10800;
+
+    private static final String NAME = "ignite";
+    private static final Integer NATIVE_PORT = 18000;
+    private static final String DRIVER_CLASS_NAME = "org.apache.ignite.IgniteJdbcThinDriver";
+    private static final String JDBC_URL_PREFIX = "jdbc:ignite:thin://";
+    private static final String TEST_QUERY = "SELECT 1";
+    private static final String databaseName = "public";
+    private static final String username = "default";
+    private static final String password = "";
+
+    public TestingIgniteContainer()
+    {
+        super(DockerImageName.parse(IGNITE_IMAGE));
+        this.withExposedPorts(HTTP_PORT, NATIVE_PORT);
```

Comment:
Is `NATIVE_PORT` really required to expose?

Reply:
Removed